server/ladders-remote.ts

```diff
@@ -149,4 +150,34 @@ export class LadderStore {
 	static async visualizeAll(username: string) {
 		return [`<tr><td><strong>Please use the official client at play.pokemonshowdown.com</strong></td></tr>`];
 	}
+
+	public calculateElo (p1: User | null, p2: User | null, score: number, formatId: string ) {
```

Comment:
```suggestion
	calculateElo(p1: User | null, p2: User | null, score: number, formatId: string) {
```
PS doesn't use `public`.